integration_tests/dbt_project.yml

```diff
@@ -1,17 +1,22 @@
 name: 'hubspot_integration_tests'
-version: '0.18.0'
+version: '0.19.0'
 
 profile: 'integration_tests'
 config-version: 2
 
 models:
-  +schema: "{{ 'hubspot_sqlw_tests' if target.name == 'databricks-sql' else 'hubspot' }}"
+  +schema: "{{ 'hubspot_sqlw_tests_2' if target.name == 'databricks-sql' else 'hubspot' }}"
   # +schema: "hubspot_{{ var('directed_schema','dev') }}" ## To be used for validation testing
 
 vars:
-  hubspot_schema: hubspot_integration_tests_57
+  hubspot_schema: hubspot_integration_tests_58
+  # hubspot__company_pass_through_columns:
+    # - name: "property_hs_all-funky-a9384-syntax"
+    #   alias: "funky_field"
+  hubspot_sales_enabled: true # enable when generating docs
```

Comment:
We should comment lines 16-17 out before merging?

Reply:
thanks! Yes will comment out the unnecessary configs